src/app/configuration/communities/community-list/community-list.component.scss

```diff
@@ -1 +1,6 @@
-/* stylelint-disable */
+.page-header {
```

Comment:
pull from development and copy paste the style here like it is at country, region, district please

Reply:
Done.